### PR TITLE
Medical - Add debuffs #VGM-523

### PR DESCRIPTION
**Pull request by veteran29**
- title

### CODE REVIEW

File: game/functions/core/client/coefficient/fn_coefficient_preInit.sqf [hunk 1]
```diff
@@ -14,3 +14,33 @@
...
+    if (isNil {_unit getVariable "vgm_c_coefficientThrowEh"}) then {
+        format ["Adding throw coef eh to: %1", _unit] call vgm_g_fnc_logDebug;
+
+        private _eh = _unit addEventHandler ["Fired", {
+            params ["_unit", "_weapon", "", "", "", "", "_projectile"];
+            if (_weapon != "Throw") exitWith {};
+
+            private _coef = _unit getVariable ["vgm_c_coefficient_throw", 1];
+            _projectile setVelocity (velocity _projectile vectorMultiply _coef);
+        }];
```
Comment (Spoffy) [inline game/functions/core/client/coefficient/fn_coefficient_preInit.sqf:38]: Does this persist after death?

File: game/functions/systems/medical/client/fn_medical_removeWound.sqf [hunk 2]
```diff
@@ -43,61 +43,61 @@ if (!(_unit call vgm_c_fnc_medical_shouldBleed)) then {
 
 ["vgm_medical_woundRemoved", _this] call para_g_fnc_event_triggerLocal;
 
-// this part needs refactoring
+// this part needs refactoring, possibly into some sort of statemachine?
```
Comment (Spoffy) [inline game/functions/systems/medical/client/fn_medical_removeWound.sqf:46]: Agreed - lots of duplication between this and addWound too. 

Fine for now (just want to get it out of the door), but we should definitely make some time to clean this up when we next touch the medical system.

**Review by Spoffy — APPROVED** [review]

File: game/functions/core/client/coefficient/fn_coefficient_preInit.sqf (hunk 1 above)
Comment (veteran29) [reply game/functions/core/client/coefficient/fn_coefficient_preInit.sqf:38]: Yeah, unit event handlers attach to "AIBrain" not to the unit itself and that "brain" moves to new unit when you respawn.

Comment (Spoffy) [reply game/functions/core/client/coefficient/fn_coefficient_preInit.sqf:38]: Ah, perfect, thanks for the info 🙂

**Pull request merged**